CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Reduced support for BitBox01
 - Fix a bug that would prevent the app to perform firmware upgrade when offline.
 - Replace sidebar with bottom navigation bar for mobile devices
+- Introduce full screen selector for mobile
```

Comment:
nit: as a user I might not understand what this means, consider mentioning "instead of dropdown"  or similar ?